messages/open.json

```diff
@@ -1,16 +1,18 @@
 {
-  "description": "open your default scratch org, or another specified org\nTo open a specific page, specify the portion of the URL after \"yourInstance.salesforce.com/\" as --path.\nFor example, specify \"--path lightning\" to open Lightning Experience, or specify \"--path /apex/YourPage\" to open a Visualforce page.\nTo generate a URL but not launch it in your browser, specify --urlonly.",
+  "description": "open your default scratch org, or another specified org\nTo open a specific page, specify the portion of the URL after \"yourInstance.salesforce.com/\" as --path.\nFor example, specify \"--path lightning\" to open Lightning Experience, or specify \"--path /apex/YourPage\" to open a Visualforce page.\nTo generate a URL but not launch it in your browser, specify --urlonly.\nTo open in a specific browser specify the app name according to your OS as \"--browser firefox",
```

Comment:
I suggest we add a bit more info to help the user figure out their browser app name. Here's an edit of your addition:

To open in a specific browser, specify the browser's app name according to your operating system.  For example, Chrome's app name is "google chrome" on macOS, "google-chrome" on Linux and "chrome" on Windows.  So to open an org in Chrome on macOS, specify --browser "google chrome". 

(I didn't add any escapes for the quotes, I'll let you do that :-)